pkg/operator/api/userconfig/apis.go

```diff
@@ -82,7 +82,7 @@ func (api *API) UserConfigStr() string {
 	sb.WriteString(fmt.Sprintf("%s: %s\n", ModelKey, yaml.UnescapeAtSymbol(api.Model)))
 
 	if api.ModelType != UnknownModelType {
-		sb.WriteString(fmt.Sprintf("%s: %s\n", ModelTypeKey, api.ModelType.String()))
+		sb.WriteString(fmt.Sprintf("%s: %s\n", ModelFormatKey, api.ModelType.String()))
```

Comment:
Should this be `api.ModelFormat`? Sorry for the rabbit hole...

Reply:
Oh no! Thanks